src/app/pages/questions/containers/questions-page.component.ts

```diff
@@ -156,7 +156,9 @@ export class QuestionsPageComponent {
       this.setNextDisabled()
 
       if (
-        this.questions[this.currentQuestion].field_type === QuestionType.timed
+        this.questions[this.currentQuestion].field_type ===
+          QuestionType.timed ||
+        this.questions[this.currentQuestion].field_type === QuestionType.audio
```

Comment:
It would perhaps be better to define a `Set` of `QuestionType`s at the top which have the previous button disabled. Then here you can check if the current field type is contained in that Set. This will make it easier to manage (if we want to add more types in the future with previous disabled) and more readable code.

Reply:
Sure I can do that, but can I do it in a separate pull request for the next release? Cause I did a lot of the refactoring in the other release and the component already has a lot of changes there.

Reply:
Ok yes Sure. Thanks.

Reply:
Fixed in #647